src/test/java/mat/server/service/jobs/CheckUserChangePasswordLimitTest.java

```diff
@@ -1,40 +1,119 @@
 package mat.server.service.jobs;
 
-import org.junit.jupiter.api.BeforeEach;
+import mat.config.MatAppContextTest;
+import mat.dao.EmailAuditLogDAO;
+import mat.dao.UserDAO;
+import mat.model.SecurityRole;
+import mat.model.Status;
+import mat.model.User;
+import mat.model.UserPassword;
+import mat.server.service.UserService;
+import org.apache.commons.lang3.time.DateUtils;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.mail.SimpleMailMessage;
 import org.springframework.mail.javamail.JavaMailSender;
 
-import mat.config.MatAppContextTest;
-import mat.dao.UserDAO;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
 
 public class CheckUserChangePasswordLimitTest extends MatAppContextTest {
     @MockBean
     UserDAO userDAO;
-    @Autowired
+    @MockBean
```

Comment:
hm, I didn't design MatAppContextTest for an extension. it was just a way to test the app context. should we introduce a parent base context tets class? :)